jupyterhub_moss/templates/option_form.html

```diff
@@ -252,13 +254,18 @@ <h4 style="text-align: center">Available resources</h4>
               id="environment_add_path"
               placeholder="/path/to/jupyter/env/bin *"
             />
+            <input
+              type="text"
+              id="environment_add_modules"
+              placeholder="List of modules (space separated)"
```

Comment:
It would be best to be consistent for `space separated` with `Extra options (space-separated)` on the same page

Either:
```suggestion
              placeholder="List of modules (space-separated)"
```
or changing `Extra options (space-separated)`.